lib/vehicle.rb

```diff
@@ -22,4 +27,22 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+  def regular?
+    @engine == :ice
+  end
+
+  def register
+    @registration_date = Date.today
+  end
+
+  def set_plate_type
+    if @engine == :ev
+      @plate_type = :ev
+    elsif Date.today.year - @year > 25
+      @plate_type = :antique
+    else
+      @plate_type = :regular
+    end
```

Comment:
I like that you created these helper methods in the Vehicle class. And good job testing them.